test/test.util.ts

```diff
@@ -50,18 +61,23 @@ describe('util', () => {
     it('should evict items older than a supplied `maxAge`', async () => {
       const maxAge = 50;
 
+      sandbox.clock = sinon.useFakeTimers();
+
       const lru = new LRUCache({capacity: 5, maxAge});
 
       lru.set('first', 1);
       lru.set('second', 2);
 
-      await new Promise(res => setTimeout(res, maxAge + 1));
+      // back to the future 🏎️
+      sandbox.clock.tick(maxAge + 1);
 
+      // just set, so should be fine
       lru.set('third', 3);
+      assert.equal(lru.get('third'), 3);
```

Comment:
Is this intended to be part of this PR?

Reply:
Accident; somehow the target wasn't `main`: #1667